api/core/agent/cot_chat_agent_runner.py

```diff
@@ -27,7 +28,7 @@ def _organize_system_prompt(self) -> SystemPromptMessage:
 
         return SystemPromptMessage(content=system_prompt)
 
-    def _organize_user_query(self, query, prompt_messages: list[PromptMessage] = None) -> list[PromptMessage]:
+    def _organize_user_query(self, query, prompt_messages: Optional[list[PromptMessage]] = None) -> list[PromptMessage]:
```

Comment:
The change introduces a potential bug where the prompt_messages parameter can be None, but the method does not handle this case. If prompt_messages is None, the code will raise an AttributeError when trying to append to it.


```suggestion
    def _organize_user_query(self, query, prompt_messages: list[PromptMessage]) -> list[PromptMessage]:
```

Reply:
SGTM. Fixed similar occurrences in `cot_chat_agent_runner.py` and `fc_agent_runner.py`